.travis.yml

```diff
@@ -7,6 +7,7 @@ before_install:
   - sudo echo "deb http://apt.openvstorage.org unstable main" | sudo tee -a /etc/apt/sources.list
   - sudo rm -rf /var/lib/apt/lists/*
   - sudo apt-get -qq update
+  - sudo apt-get install -y --force-yes volumedriver-server
```

Comment:
Please use volumedriver-no-dedup-server as this is now the default for unstable
- without MD5 support, it allows for a smaller memory footprint